pack3d/pack3d.go

```diff
@@ -197,7 +201,7 @@ func Pack(config *Config) (*PackingOutput, error) {
 		X20: 0, X21: 0, X22: 0, X23: 0,
 		X30: 0, X31: 0, X32: 0, X33: 0,
 	}
-	timeLimit = 10
+	timeLimit = 20 // second
```

Comment:
@ted-authentise @AyoubAuthentise I was really hoping youse or Tyler (not @ ing Tyler, he is busy on a deadline ) would have caught this, and asked why or for some docs. I doubled how long this thing works, and didn't add notes, or docs, or aything on it 😢 😢 😭 . restoring to 10 second  in my new PR